Composer/packages/types/src/publish.ts

```diff
@@ -170,6 +170,10 @@ export type PublishProfile = {
       endpoint: string;
       region: string;
     };
+    qna?: {
```

Comment:
To match: extensions\azurePublish\src\node\schema.ts

I think we need to add:
tenantId
language
luisResource
botName
resourceGroup
subscriptionId
region
settings/qna/endpoint